test/cljs/rems/test/administration/form.cljs

```diff
@@ -8,7 +8,6 @@
 (use-fixtures :each isolate-re-frame-state)
 
 (defn reset-form []
-  (stub-re-frame-effect ::f/fetch-form-items)
```

Comment:
Why this away?

Reply:
There is no actual event called `fetch-form-items` in the `form.cljs` and it only creates unnecessary warnings when running test code.